qdrant-landing/content/documentation/tutorials/async-api.md

```diff
@@ -0,0 +1,362 @@
+how to do it.
+
+## When the usage of async API is justified?
+
+If the application you are writing will never support multiple users at once, for example, it is a script run once daily, then there 
+is no need to use async API. But if you are writing a web service that multiple users will use simultaneously, you shouldn't be 
+blocking the threads of the web server as it limits the number of concurrent requests it can handle. In this case, you should use 
+the async API.
+
+Modern web frameworks like [FastAPI](https://fastapi.tiangolo.com/) and [Quart](https://quart.palletsprojects.com/en/latest/) support 
+async API out of the box. Asynchronous code cannot be incorporated into an existing synchronous codebase, so it's better to choose 
+the proper tool from the start.
+
+## How to use async API?
+
+As usual, calling any method of Qdrant requires establishing a connection to the server. We need to create an instance of `QdrantClient`
+that will act as a gateway. If you want to do it locally, please make sure Qdrant server is running. If it's not, then you can launch it
+in a Docker container:
+
+```bash
+docker run -p "6333:6333" -p "6334:6334" qdrant/qdrant:v1.4.0
+```
+
+Alternatively, you can also use a [Qdrant Cloud](https://cloud.qdrant.io/) cluster. In this case, you need to pass the API key to the
+client constructor. No matter which way you choose, the `QdrantClient` instance has to be created:
+
+```python
+from qdrant_client import QdrantClient
+
+client = QdrantClient("localhost")
+# If you use Qdrant Cloud, then pass the API key to the constructor:
+# client = QdrantClient("https://your-cluster-url.cloud.com", api_key="your-api-key")
+```
+
+The top-level client exposes just the synchronous methods. We can access the autogenerated async clients and call the underlying methods.
+It is possible to use asynchronous HTTP API by using `qdrant_client.http.api_client.AsyncApiClient`. However, the gRPC API is much more 
+efficient and is the recommended way of using Qdrant asynchronously.
+
+### gRPC API
+
+Every instance of `QdrantClient` has two properties we are going to use: `async_grpc_collections` and `async_grpc_points`. They expose
+the autogenerated clients for collections and points respectively. 
+
+The gRPC client uses type definitions for all the interactions with the server. They are autogenerated from the source code of Qdrant
+server, and we have to import them if we want to call any of the available methods.
+
+```python
+from qdrant_client import grpc
+```
+
+If your IDE does not support autocompletion for the autogenerated clients, you can refer to the documentation and [see the available
+fields for each of them](https://github.com/qdrant/qdrant/blob/master/docs/grpc/docs.md).
+
+#### Creating a collection
+
+Let's create a collection and add some points to it.
+
+```python
+response = await client.async_grpc_collections.Create(
+    grpc.CreateCollection(
```

Comment:
Am I right in thinking that async clients are basically thin wrappers around all the synchronous calls? How much can I apply my knowledge of the synch APIs to work with the async APIs

Reply:
OK good to know